apps/api/lib/api/web/router.ex

```diff
@@ -2,6 +2,8 @@ defmodule API.Web.Router do
   use API.Web, :router
 
   pipeline :api do
+    plug Guardian.Plug.VerifyHeader
+    plug Guardian.Plug.LoadResource
```

Comment:
Is this working? I tried putting this in for the refresh token endpoint and couldnt get it working